File: back-end/src/main/java/com/espto/espto/dto/EventDashboard.java
package com.espto.espto.dto;

import com.espto.espto.enums.EsporteTipo;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Builder
@Getter
@Setter
public class EventDashboard {

    private Long id;

    private Long idUser;

    private EsporteTipo sportType;

    private Integer amountParticipants;

    private Integer amountActiveParticipants;

    private String local;

    private String nextSchedule;

}

File: front-end/src/app/demo/api/event-dashboard.ts
import { EsporteTipo } from './evento';

export class EventDashboard {
    id: number;
    idUser: number;
    sportType: EsporteTipo;
    amountParticipants: number;
    amountActiveParticipants: number;
    local: string;
    nextSchedule: string;
}
